assets/js/types.ts

```diff
@@ -40,6 +40,7 @@ export type OriginalEventInfo = {
   presenters: string[];
   start_date?: string;
   uuid: string;
+  moderator_email?: string;
```

Comment:
when is this field undefined?

Reply:
When you make a request for a specific webinar. Because we are adding `moderator_email` only when we are requesting a list of webinars/recordings.